ov/ov_test.go

```diff
@@ -33,22 +33,22 @@ type OVTest struct {
 }
 
 // get Environment
-func (ot *OVTest) GetEnvironment() {
+func (ot *OVTest) GetEnvironment(env string) {
 	if os.Getenv("ONEVIEW_TEST_ENV") != "" {
 		ot.Env = os.Getenv("ONEVIEW_TEST_ENV")
 		return
 	}
-	ot.Env = "dev"
+	ot.Env = env
```

Comment:
Trying to understand why we want this.  Could you explain the challenge your facing with not letting environment represent a default globally for all test here and what use case we're solving by allowing every test case to define their own default environment value?    My thoughts on environment was that we could add progressively complex testing for things like a test and production environment in the future, but in this scenario, I'm thinking we might just be moving a default that should be located here to all other test case calls.... wanted to get your take on this.